jmclient/jmclient/configure.py

```diff
@@ -329,6 +329,13 @@ def jm_single():
 # for payjoins to hidden service endpoints, the socks5 configuration:
 onion_socks5_host = localhost
 onion_socks5_port = 9050
+
+# for payjoin hidden service creation, the tor control configuration:
+control_host = localhost
+# or, to use a UNIX socket
+# control_host = unix:/var/run/tor/control
+control_port = 9051
```

Comment:
I would suggest renaming these to `tor_control_host` and `tor_control_port`.

Reply:
Fixed in fde39a7